portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,30 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments = new ArrayList<String>();
```

Comment:
Instead of instantiating this variable here, can we directly instantiate it in the method `doGet`? Its not used anywhere else.

Reply:
On a similar note, could this just be a static final string array, given that it always has the same values?

Reply:
made comments static final but kept it outside of the method because the static variable cannot be declared inside the method